src/tike/random.py

```diff
@@ -25,31 +25,43 @@ def wobbly_center(obs, k, xp=cp):
     Parameters
     ----------
     obs : (M, N) array_like
-        The N dimensional population of N samples that needs to be clustered.
-    k : int < M
-        The number of clusters in which to divide M.
+        The N dimensional population of M samples that needs to be clustered.
+    k : int (0..M]
```

Comment:
This would be easier to read if "k" was something like "number_of_clusters" and "N" was something like "number_of_samples".

Reply:
Renamed k, to num_clusters, but left N and M alone because they are only used in the docs.